### ISSUE TITLE
domain field "ip_address" should not be required

### ISSUE THREAD

**Issue by marcoebbinghaus** [terraform-providers/terraform-provider-digitalocean#134]
The DigitalOcean v2 api says ip_address-field for domain resources is not required (https://developers.digitalocean.com/documentation/v2/#create-a-new-domain), but in terraform that field IS required (https://www.terraform.io/docs/providers/do/r/domain.html).

### Terraform Version
v0.11.8

### Affected Resource(s)
digitalocean_domain

### Terraform Configuration Files
```
resource "digitalocean_domain" "default" {
  name       = "www.example.com"
}
```
### Expected Behavior
ip_address is not a required field

### Actual Behavior
ip_address is a required field

### Steps to Reproduce
1. `terraform plan` with a domain resource without ip_address-field set

**Comment by andrewsomething** [comment]
A change allowing this landed just last week in #122 and will be available in the next release.  Thanks for the report!

**Comment by marcoebbinghaus** [comment]
oh missed that, cool..thank you!
